adam-core/src/test/scala/org/bdgenomics/adam/rdd/feature/FeatureRDDSuite.scala

```diff
@@ -586,7 +586,7 @@ class FeatureRDDSuite extends ADAMFunSuite with TypeCheckedTripleEquals {
     // we can't guarantee that we get exactly the number of partitions requested,
     // we get close though
     assert(jRdd.rdd.partitions.length === 1)
-    assert(jRdd0.rdd.partitions.length === 5)
+    assert(jRdd0.rdd.partitions.length === 4)
```

Comment:
Yeah, this change is expected. In the old region join code, you could "suggest" a number of partitions (which would then suggest a base pair size for the partition), but you weren't guaranteed to get that exact amount. In the new code, when you ask for 4 partitions, you get 4 partitions.